src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -182,22 +184,79 @@ public sealed class BaseFeature : FullNodeFeature
             this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
         }
 
+        public static void SetRewindFlag(NodeSettings nodeSettings, int height)
+        {
+            string[] configLines = File.ReadAllLines(nodeSettings.ConfigurationFile);
+
+            if (configLines.Any(c => c.Contains(BaseFeature.RewindFlag)))
```

Comment:
Maybe `StartsWith` would be more robust? Anyone could add another config key, or even a comment, that would meet this condition